torch/csrc/jit/register_prim_ops_c10.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/ATen.h>
+#include <ATen/core/stack.h>
+
+using Stack = std::vector<c10::IValue>;
+using torch::jit::peek;
+using torch::jit::drop;
+using torch::jit::pack;
+using torch::jit::push;
+using torch::jit::pop;
+using at::Tensor;
+using at::Scalar;
+using c10::IValue;
+
+static auto registry_prim = torch::RegisterOperators().op("aten::Int.Tensor(Tensor a) -> int",
+  torch::RegisterOperators::options().kernel(c10::TensorTypeId::CPUTensorId,
+  [](at::Tensor a) -> int64_t {
+    return a.item<int64_t>();
+})
+).op(  "aten::Int.bool(bool a) -> int",
+  torch::RegisterOperators::options().catchAllKernel(
+  [](bool b) -> int64_t {
+    return (int64_t)b;
```

Comment:
nit: `static_cast` is better than C-style casts. This was already wrong in the old code, but let's fix it since we touch this code anyways.